### PR TITLE
Makes the plugin to load ASAP after Worldguard

### PR DESCRIPTION
**Pull request by LunarcatOwO**

### CODE REVIEW

**Review by iseal56 — CHANGES_REQUESTED** [review]
The version needs to stay in format to avoid the update checker bugging out, so make it 3.5.0.2

File: src/main/resources/plugin.yml [hunk 1]
```diff
@@ -1,8 +1,10 @@
-name: PowerGems
+name: PowerGems-LunarsPatch
```
Comment (iseal56) [inline src/main/resources/plugin.yml:1]: You changed the plugin name.

**Review by iseal56 — CHANGES_REQUESTED** [review]

File: src/main/resources/plugin.yml (hunk 1 above)
Comment (LunarcatOwO) [reply src/main/resources/plugin.yml:1]: yes. lol

**Review by iseal56 — APPROVED** [review]
Has this been tested to solve the issue? If so, push.

**Pull request merged**